### PR TITLE
Add mobile sidebar.

### LINKED ISSUE #14: Frontend: Create mobile version of the header

### PR DESCRIPTION
**Pull request by farebord**
Closes #14

### CODE REVIEW

**Comment by VargaElod23** [comment]
Is it just me or the header elements are squashed now? 
image

File: src/components/header/Header.stories.tsx [hunk 1]
```diff
@@ -7,24 +7,78 @@ export default {
...
+MobileConnected.args = {
+  variant: "mobile",
+  headerElements: [
+    HeaderValues.HypeFarming,
+    HeaderValues.HypePool,
+    HeaderValues.Redeem,
+  ],
+  status: "connected",
+  onConnect: () => {},
+  account: "0x6e3AA...82aFB",
```
Comment (VargaElod23) [inline src/components/header/Header.stories.tsx:22]: Here we should inject the full account and make the component standardize it with the MM stnadard: `5 chars ... 4 chars`

**Review by VargaElod23 — CHANGES_REQUESTED** [review]
Code looks good, 1 small suggestion and please take a look at the header element styling issue on the comment above.

File: src/components/header/Header.stories.tsx (hunk 1 above)
Comment (farebord) [reply src/components/header/Header.stories.tsx:22]: Ohh I see! I thought it came as 5chars ... 4 chars as default. @VargaElod23 Should it be `0x12345...1234` ?

Comment (Elod23) [reply src/components/header/Header.stories.tsx:22]: 0x123...4567

**Review by VargaElod23 — APPROVED** [review]
👍🏼

**Pull request merged**